components/list/README.md

```diff
@@ -56,100 +98,285 @@ The `grid` attribute will enable a table-like keyboard grid that allows a user t
+<d2l-list>
+  <d2l-list-item href="http://www.d2l.com" selectable key="3">
+    <img slot="illustration" src="https://s.brightspace.com/course-images/images/63b162ab-b582-4bf9-8c1d-1dad04714121/tile-high-density-max-size.jpg" />
+    <d2l-list-item-content>
+      <div>Geomorphology and GIS </div>
+      <div slot="supporting-info">This course explores the geological processes of the Earth's interior and surface. These include volcanism, earthquakes, mountain...</div>
+    </d2l-list-item-content>
+    <div slot="actions">
+      <d2l-button-icon text="My Button" icon="tier1:preview"></d2l-button-icon>
+    </div>
+  </d2l-list-item>
+</d2l-list>
 ```
 
-**Properties:**
+<!-- docs: start hidden content -->
+### Properties
 
 | Property | Type | Description |
-|--|--|--|
+|---|---|---|
 | `href` | String | Address of item link if navigable |
 
-**Events**
+### Events
 
 - `d2l-list-item-link-click`: dispatched when the item's primary link action is clicked
+<!-- docs: end hidden content -->
 
-## d2l-list-item-button
+### Breakpoints Property
 
-The `d2l-list-item-button` provides the same functionality as `d2l-list-item` except with button semantics for its primary action. It extends `ListItemButtonMixin` and `ListItemMixin` and has all the same use cases as the mixin.
-
-![List](./screenshots/list-item.png?raw=true)
-
-```html
-<d2l-list-item-button breakpoints="array"
-  @d2l-list-item-button-click="..."
-  key="item1"
-  label="some label"
-  selectable
-  selected
-  disabled>
-  <img src="..." slot="illustration">
-  <div>...</div>
-  <div slot="actions">
-    <d2l-button-icon ...></d2l-button-icon>
-    <d2l-button-icon ...></d2l-button-icon>
-  </div>
-</d2l-list-item-button>
-```
-
-**Events**
-
-- `d2l-list-item-button-click`: dispatched when the item's primary button action is clicked
+- `breakpoints` (Array): Breakpoints for responsiveness (`[842, 636, 580, 0]`), in pixels. There are four different breakpoints and only the four largest breakpoints will be used. If less breakpoints are used, then skip a middle breakpoint so that the first and last breakpoints will map to the largest and smallest layouts.
+  - Breakpoint 0
+    - Image: max dimensions: `width: 90px` and `height: 52px` and has `18px margin` from the main content;
+    - default break: `x < 580px` where `x` is the width of the component.
+  - Breakpoint 1
+    - Image: max dimensions: `width: 120px` and `height: 71px` and has `20px margin` from the main content;
+    - default break: `581px < x < 636px` where `x` is the width of the component.
+  - Breakpoint 2
+    - Image: max dimensions: `width: 180px` and `height: 102px` and has `20px margin` from the main content;
+    - default break: `637px < x < 842px`  where `x` is the width of the component.
+  - Breakpoint 3
+    - Image: max dimensions: `width: 216px` and `height: 120px` and has `20px margin` from the main content;
+    - default break: `843px < x`  where `x` is the width of the component.
 
 ## ListItemMixin
```

Comment:
I tried to get this section to be a working demo example but couldn't quite get it to work so leaving mostly as is for now.